evm/ts-scripts/shell/verify-governance-contract.sh

```
@@ -0,0 +1,60 @@
+if test -z "$ENV"; then
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)